seccon23-rev-perfect-blu/index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: "seccon23 rev/Perfect Blu"
+publishDate: "18 Sep 2023"
+description: "Author: es3n1n"
+tags: ["rev", "seccon23"]
+---
+
+
+#### Description
+
+Perfect Blu (135 pt)
+
+No, I'm real!
+
+[perfect-blu.tar.gz](https://score.quals.seccon.jp/api/download?key=quals2023%2FPerfect_Blu%2Fperfect-blu.tar.gz) 367b6ed67dda0afbbc975ee70ee946b4c7bf9268
+
+
+#### Overview
+
+Once I opened the downloaded `.tar.gz` file I observed that there was a `.iso` file inside and once I mounted it I saw that it had a file structure that looked 
+like the fs structure of a DVD. So what I did is I dragged this `.iso` file to the VLC and was very surprised.
+
+![1](./img/1.png)
+
+As you probably already figured, this task is a DVD image that has an interactive menu where you need to enter the flag, press on the `CHECK` button and, depending on the flag it would, either show a `SUCCESS` or a `WRONG...` message.
+
+#### Analysing
+
+_I'm gonna be completely honest, I never had any experience with all this DVD stuff before, so I had to spend a couple of hours googling how to analyse this stuff first._
+
+There a multiple types of DVD menus that I'm aware of:
+- Java menus
+- IGS Menus
+
+What I did first is I checked for any .jar files or just something that could remind me of Java, but I didn't find anything and it meant that I was dealing with the IGS menu.
+
+The first step in IGS menu analysis is to download [BDEdit](https://bdedit.pel.hu/) and open up our mounted `BDMV\index.bdmv`
+
+![2](./img/2.png)
+
+Looks scary, I know. What I did is, I checked the clip info for the `und` streams. To do this I clicked on the `CLIPINFO` menu and found the `und` stream in the GUI that contained our buttons logic.
```

Comment:
Too many "What I did". Use some other introductory ffs.

also what the heck are und streams? your readers likely don't know that (i dont)

_found the `und` stream in the GUI that contained our buttons logic_ - it's not the GUI that contains the logic, but the und stream. wording could be improved. also you explain that it contains the logic in the next sentence, so it can be omitted here.

```
Looks scary, I know. 
First of all, I checked the clip info for the `und` streams. `und` streams are...

To do that, I clicked on the `CLIPINFO` menu and found the `und` stream in the GUI that appeared:
```